apps/percona/workload/tpcc.conf

```diff
@@ -0,0 +1,9 @@
+{
+  "db_user": "root",
+  "db_password": "k8sDem0",
```

Comment:
Can we have tasks in test.yml to update these with envs obtained in run-litmus.yml